Description: Adapt SCALA_HOME and TOOL_CLASSPATH to the package layout.
Origin: vendor
Forwarded: not-needed
Author: Frank S. Thomas <fst@debian.org>

Index: b/src/compiler/scala/tools/ant/templates/tool-unix.tmpl
===================================================================
--- a/src/compiler/scala/tools/ant/templates/tool-unix.tmpl
+++ b/src/compiler/scala/tools/ant/templates/tool-unix.tmpl
@@ -48,23 +48,7 @@
 esac
 
 # Finding the root folder for this Scala distribution
-SOURCE=$0;
-SCRIPT=`basename "$SOURCE"`;
-while [ -h "$SOURCE" ]; do
-    SCRIPT=`basename "$SOURCE"`;
-    LOOKUP=`ls -ld "$SOURCE"`;
-    TARGET=`expr "$LOOKUP" : '.*-> \(.*\)$'`;
-    if expr "${TARGET:-.}/" : '/.*/$' > /dev/null; then
-        SOURCE=${TARGET:-.};
-    else
-        SOURCE=`dirname "$SOURCE"`/${TARGET:-.};
-    fi;
-done;
-
-# see #2092
-SCALA_HOME=`dirname "$SOURCE"`
-SCALA_HOME=`cd "$SCALA_HOME"; pwd -P`
-SCALA_HOME=`cd "$SCALA_HOME"/..; pwd`
+SCALA_HOME="/usr/share/java"
 
 # Remove spaces from SCALA_HOME on windows
 if $cygwin; then
@@ -73,16 +57,7 @@
 fi
 
 # Constructing the extension classpath
-TOOL_CLASSPATH="@classpath@"
-if [ -z "$TOOL_CLASSPATH" ] ; then
-    for ext in "$SCALA_HOME"/lib/* ; do
-        if [ -z "$TOOL_CLASSPATH" ] ; then
-            TOOL_CLASSPATH="$ext"
-        else
-            TOOL_CLASSPATH="$TOOL_CLASSPATH:$ext"
-        fi
-    done
-fi
+TOOL_CLASSPATH="$SCALA_HOME/scala-compiler.jar:$SCALA_HOME/scala-library.jar:$SCALA_HOME/scalap.jar:$SCALA_HOME/scala/jline.jar"
 
 CYGWIN_JLINE_TERMINAL=
 if $cygwin; then
